pilbox/app.py

```diff
@@ -209,7 +210,7 @@ def fetch_image(self):
             raise tornado.gen.Return(resp)
         except (socket.gaierror, tornado.httpclient.HTTPError) as e:
             logger.warn("Fetch error for %s: %s",
-                        self.get_argument("url"),
+                        url,
```

Comment:
why was this changed?

Reply:
Check commit message:
`Will make debugging fetch issues a lot easier`

It was having a hell of a time debugging an issue w/ implicit_base_url earlier yesterday because I never got any log information that contained the base url in it, so I could not be 100% positive whether the setting was actually used.

Reply:
Looks like the old code was incorrect